conda_smithy/feedstock_content/.github/ISSUE_TEMPLATE.md

```diff
@@ -1,46 +1,25 @@
+<!--
 Thanks for reporting your issue.
-
-
 Please fill out the sections below.
+-->
+Issue
 
-
-Example (code snippet, shell command, or screenshot):
-
-<details>
-
-
-
-</details>
-
-<br>
-Environment (`conda env export`):
-<br>
-<br>
```

Comment:
This was mainly here as there are some annoying bugs in GitHub when rendering several `details` sections. So if we only want one, that should be fine. Just figured I'd mention it to provide context.

Reply:
I checked both before sending the PR. Maybe github fixed their bugs